README.md

```diff
@@ -28,13 +28,13 @@ href="NEWS.md">Release&nbsp;Notes</a>]</h3>
 Labwc stands for Lab Wayland Compositor, where lab can mean any of the
 following:
 
-- Lightweight And Box-inspired
+- lightweight and box-inspired
 - sense of experimentation and treading new ground
 - inspired by BunsenLabs and ArchLabs
 - your favorite pet
 
-Labwc is a [wlroots]-based window-stacking compositor for [wayland], inspired
-by [openbox].
+Labwc is a [wlroots]-based window-stacking compositor for [Wayland], inspired
+by [Openbox].
```

Comment:
As these are links (defined at the bottom) I am not sure if the casing is relevant or not. At least currently with the GitHub rendering engine it doesn't seem to matter but I suggest changing the link targets to this new casing as well.

Reply:
yep, looks good :)